index.rst

```diff
@@ -221,5 +246,128 @@ Banging on the side of the strut to excite some kind of bending mode was less su
+
+		
+.. image:: ./_static/Dome_Setup_28Nov22.jpg
+
+**Setup on 28-Nov-22**
+
+
+These images show the anemometer readings, the accelerometer readings, and the mount encoder data as the fan speed is ramped up:
+
+
+Measurements with Fan Speed at 0.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112800003.png
+
+Measurements with Fan Speed at 10.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112800008.png
+
+Measurements with Fan Speed at 20.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112800013.png
+
+Measurements with Fan Speed at 30.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112800018.png
+
+Measurements with Fan Speed at 40.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112800023.png
+
+Measurements with Fan Speed at 50.0 Hz
+------------------------------------------------------
+
+   .. image:: ./_static/Accel_Anemom_Mount_2022112800028.png
+
+
+Summary of measurements
+-----------------------------------------------
+
+   .. image:: ./_static/Anemom_vs_Fan_Speed_28Nov22.png
+	      
+   .. image:: ./_static/Joint_Ane_Accel_Mount_Tests_28Nov22.png
+
+This is a summary of the measurements taken.  The total accelerometer power, and the mount jitter all ramp up as the fan speed is increased. All of the anemometer standard deviations increase as the fan speed is ramped, indicating increased turbulence.  In addition, the Uz mean speed also increases.  The anemometer readings show much more scatter than expected.
+
+
+Conclusions
+=================================
+
+This report is somewhat of a "data dump", with a lot of data that is difficult to interpret.  One consistent finding in all of the tests is that the mount jitter increases steadily as the fan speed is increased.  For this reason, it is not recommended to go above 20 Hz on the frequency meter when observing.  We should discontinue use of the hand written (25%-50%-75%) markings on the speed adjustment and use instead the frequency meter, which is much more accurate.  The accelerometer and anemometer readings are qualitatively consistent with increasing wind turbulence and mount shake as the wind speed is increased, but it is difficult to draw any quantitative conclusions from these readings. One clear result is that the anemometer standard deviations increase consistently as the fan speed is increased, indicating increased turbulence.  I am open to suggestions as to further measurements or analysis.
```

Comment:
D'Oh you answered my question above about the hand written parts here...

Reply:
I think I've made all of the requested changes.  Merging.